templates/schedule/item.html

```diff
@@ -135,8 +135,16 @@ <h4>Video</h4>
     <iframe src="{{proposal.youtube_url|replace('youtube.com/watch?v=', 'youtube.com/embed/')}}"
         width="100%" height="500px" frameborder="0" 
         allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-    <p>View this video <a href="{{ proposal.youtube_url }}">on YouTube</a>.</p>
   {% endif %}
+  <p>Direct links to the host(s) of this video recording can be found below:</p>
```

Comment:
This sentence has a verbose/technical tone to it - it doesn't read naturally.

Originally I thought about simply using a section heading of `External Links` -- but I didn't want to imply that the embedded video on the page is hosted on the EMF website (because it isn't).